File: UNIDAD 5/JAVA/U5_P7.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u5_p7;
import java.util.Scanner;
/**
 *
 * @author fatim
 */
public class U5_P7 {

    public static void main(String[] args) {
       Scanner L = new Scanner(System.in);
       double ST, T, I, M;
    
        System.out.print("Ingrese el monto a calcular: $");
        M = L.nextDouble();

        I = CalcularIVA(M);

        T = CalcularTotal(M, I);
        
        ST= CalcularSubTotal(T, I);

      
        
        System.out.println("IVA (16%) es: $" + I);
        System.out.println("Total es: $" + T);
        System.out.println("Subtotal es: $" + ST);

    }

    public static double CalcularIVA(double M) {
        double PorI = 0.16;
        return M * PorI;
    }

    public static double CalcularTotal(double M, double I) {
        return M + I;
    }
    public static double CalcularSubTotal(double T, double I) {
        return T - I;
    }
}

File: UNIDAD 5/C#/U5_P7.cs
using System;

namespace U5_P7
{
    internal class U5_P7
    {
        static void Main(string[] args)
        {
            Console.Write("Ingrese el monto a calcular: $");
            double M = Convert.ToDouble(Console.ReadLine());

            double I = CalcularIVA(M);
            double T = CalcularTotal(M, I);
            double ST = CalcularSubTotal(T, I);

            Console.WriteLine("IVA (16%) es: $" + I);
            Console.WriteLine("Total es: $" + T);
            Console.WriteLine("Subtotal es: $" + ST);
        }

        public static double CalcularIVA(double M)
        {
            double PorI = 0.16;
            return M * PorI;
        }

        public static double CalcularTotal(double M, double I)
        {
            return M + I;
        }

        public static double CalcularSubTotal(double T, double I)
        {
            return T - I;
        }
    }

}